torch/testing/_internal/common_methods_invocations.py

```diff
@@ -817,7 +829,15 @@ def __post_init__(self):
             inplace_name = self.name + "_"
             self.inplace_variant = getattr(torch.Tensor, inplace_name, None)
 
-        self.operator_variant = getattr(operator, self.name, None)
+        if self.operator_variant is _NOTHING:
+            self.operator_variant = getattr(operator, self.name, None)
+
+        if self.inplace_operator_variant is _NOTHING:
+            if self.inplace_variant is not None:
```

Comment:
@mruberry This is the logic I think may warrant additional explanation.

Reply:
Sounds great -- it's hard to have too many comments